<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-12-22T21:18:06 SGT" name="Default suite" finished-at="2023-12-22T21:18:24 SGT" duration-ms="17230">
    <groups>
      <group name="SmokeSuite">
        <method signature="CreateContact_Assert.CreateContactWithMandatoryField()[pri:0, instance:contactsTests.CreateContact_Assert@74fe5c40]" name="CreateContactWithMandatoryField" class="contactsTests.CreateContact_Assert"/>
      </group> <!-- SmokeSuite -->
    </groups>
    <test started-at="2023-12-22T21:18:06 SGT" name="Default test" finished-at="2023-12-22T21:18:24 SGT" duration-ms="17230">
      <class name="contactsTests.CreateContact_Assert">
        <test-method is-config="true" signature="bsConfig()[pri:0, instance:contactsTests.CreateContact_Assert@74fe5c40]" started-at="2023-12-22T21:18:06 SGT" name="bsConfig" finished-at="2023-12-22T21:18:06 SGT" duration-ms="5" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bsConfig -->
        <test-method is-config="true" signature="bcConfig()[pri:0, instance:contactsTests.CreateContact_Assert@74fe5c40]" started-at="2023-12-22T21:18:06 SGT" name="bcConfig" finished-at="2023-12-22T21:18:09 SGT" duration-ms="2585" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bcConfig -->
        <test-method is-config="true" signature="bmConfig()[pri:0, instance:contactsTests.CreateContact_Assert@74fe5c40]" started-at="2023-12-22T21:18:09 SGT" name="bmConfig" finished-at="2023-12-22T21:18:12 SGT" duration-ms="2866" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bmConfig -->
        <test-method signature="CreateContactWithMandatoryField()[pri:0, instance:contactsTests.CreateContact_Assert@74fe5c40]" started-at="2023-12-22T21:18:12 SGT" name="CreateContactWithMandatoryField" finished-at="2023-12-22T21:18:21 SGT" duration-ms="9227" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at contactsTests.CreateContact_Assert.CreateContactWithMandatoryField(CreateContact_Assert.java:33)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreateContactWithMandatoryField -->
        <test-method is-config="true" signature="amConfig()[pri:0, instance:contactsTests.CreateContact_Assert@74fe5c40]" started-at="2023-12-22T21:18:21 SGT" name="amConfig" finished-at="2023-12-22T21:18:23 SGT" duration-ms="2356" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- amConfig -->
        <test-method is-config="true" signature="acConfig()[pri:0, instance:contactsTests.CreateContact_Assert@74fe5c40]" started-at="2023-12-22T21:18:23 SGT" name="acConfig" finished-at="2023-12-22T21:18:24 SGT" duration-ms="175" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- acConfig -->
        <test-method is-config="true" signature="asConfig()[pri:0, instance:contactsTests.CreateContact_Assert@74fe5c40]" started-at="2023-12-22T21:18:24 SGT" name="asConfig" finished-at="2023-12-22T21:18:24 SGT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- asConfig -->
      </class> <!-- contactsTests.CreateContact_Assert -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
